### DISCUSSION TITLE
Vite + AWS

### DISCUSSION THREAD

**Discussion by THOUSAND-SKY** [remix-run/remix#8399] [Q&A]
Has anybody had luck with vite on aws? `remix vite:build` seems to ignore the `server.ts` and I don't see an option to make that the entrypoint. I'm not sure how I'd plug in the `architect` (though I'm using CDK) request handler creator into `entrypoint.server.ts` or any other file for that matter: where would I get the `ServerBuild`?

**Comment by GautierBlandin** [comment]
Any update on this ?

**Reply by HerrBertling** [reply]
There's this other discussion that might have a fix (my pipeline is still running, so can't tell yet 😄 )

https://github.com/remix-run/remix/discussions/8836#discussioncomment-9795687
